Makefile

```diff
@@ -24,11 +24,17 @@ fmt-check:
 ci: fmt-check test
 
 OPA_VER=v0.39.0
-OPA_URL=https://openpolicyagent.org/downloads/$(OPA_VER)/opa_linux_amd64_static
+OPA_FILE=opa_linux_amd64_static
+OPA_URL=https://openpolicyagent.org/downloads/$(OPA_VER)/$(OPA_FILE)
+OPA_SHA_FILE=$(OPA_FILE).sha256
+OPA_SHA_URL=$(OPA_URL).sha256
```

Comment:
I was thinking more of including the checkusm in git so we protect ourselves against `openpolicyagent.org` being compromised. The current version wouldn't detect if someone replaced the opa binary with some malicious binary and also replace the digest. It would help in detecting a corrupt file due to download issues, though.

Reply:
Ah yeah ok. Let me try again.